### ISSUE TITLE
Evaluation measure remains after several iterations, epochs...What could be the reason?

### ISSUE THREAD

**Issue by chowkamlee81** [apache/incubator-mxnet#8060]
Iam trying to enhance semantic segmentation results  by replacing resnet 101 by resnet152. However
training loss remains same even after several epochs, iterations. Kindly suggest what could be the reason?

**Comment by eric-haibin-lin** [comment]
Did you adjust the learning rate after it's "fixed" for a while?

**Comment by ChidanandKumarKS** [comment]
No didn't change the learning rate. I retained the old framework which was used and only change I did replace resnetr 101 with resnet152 and  probing the weights at  appropriate places resnet152. Kindly suggest how to go ahead

**Comment by chowkamlee81** [comment]
@eric-haibin-lin do you have any updates on the above issue . Since i didn't change any learning rate. But when i keep the same learning rate, resnet-101 model works fine and loss continously decreases exponentially. But the same behavior doesn't get replicated with resnet-152 model as loss curves remains fixed. Can u kindly provide some insights into this issue. Requesting you to suggest some solution

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by rahul003** [comment]
@cjolivier01 Could you help label this with Question and close it.

@chowkamlee81 If you are still facing this, please ask on our discussion forum https://discuss.mxnet.io/

**Comment by sandeep-krishnamurthy** [comment]
Please reopen if you cannot find answer for your questions at https://discuss.mxnet.io/
